migrate.go

```diff
@@ -275,7 +275,8 @@ func findMigrations(dir http.FileSystem) ([]*Migration, error) {
 }
 
 func migrationFromFile(dir http.FileSystem, info os.FileInfo) (*Migration, error) {
-	file, err := dir.Open(info.Name())
+	path := fmt.Sprintf("/%s", info.Name())
```

Comment:
Should there be a TrimPrefix here to avoid a double slash?

```suggestion
	path := fmt.Sprintf("/%s", strings.TrimPrefix(info.Name(), "/"))
```